supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -143,6 +144,29 @@ $ oc describe node __<node_name>__
 ----
 ====
 
+.Example AsciiDoc: User-replaced value in an output example
+
+----
+# *nmcli connection show __<profile_name>__*
+connection.id:              __<profile_name>__
+connection.uuid:            b6cdfa1c-e4ad-46e5-af8b-a75f06b79f76
+connection.type:            802-3-ethernet
+connection.interface-name:  enp7s0
+----
+
+This renders as:
+
+====
+[subs="+quotes"]
+----
+# nmcli connection show __<profile_name>__
```

Comment:
On OpenShift, we have a rule where we separate commands from output. This helps with being able to use the "copy" button to copy the command.

This is the first time we're introducing this (command+output) to the SSG as far as I'm aware, so I don't want to introduce it this way if we really should be separating them. 

Do others separate command from output, or keep them in the same block?

Reply:
I like SSG enhancement as currently proposed by this pull request.

Using separate blocks for command and output would require an additional intro sentence for a output block, visually increases the length of a procedure, and doesn't look nice (just my opinion).

In RHEL, most things happen on the command line, and we have commands in almost every step of a procedure. Sure, in most cases we only show the command without output, but separating the command from the output in existing docs would still be a lot of extra work.

I understand the argument of copy/paste, but I think this should be solved by JavaScript on the Customer Portal. For example, only the bold text (command) should be copied to the clipboard, when you click the copy button.

Reply:
We actually don't require another sentence introducing it. We use the heading of ".Example output", and include it right after the code block. 

An example of that is here: https://docs.openshift.com/container-platform/4.13/security/audit-log-view.html#nodes-nodes-audit-log-basic-viewing_audit-log-view

Or how it looks on the portal: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/security_and_compliance/audit-log-view#nodes-nodes-audit-log-basic-viewing_audit-log-view

It could be just because I'm used to it, but I prefer it this way, both because of the ability to use the "Copy" button and just get the command, but also visually so you can distinguish the command from the output. I'm seeing here that the command looks like it's trying to be bolded, which would help visually distinguish, but again I'm not sure if that's what people normally do.

But really, I'm not trying to debate which method is better or which we should be doing. Because that's not what we're trying to explicitly add guidance for here. But we are going to be implicitly adding guidance for however we do this. If we add it in here in the same block, and bold the command, then people are going to take that as guidance for how they should be doing it.

(Also FYI @sbmetz that it doesn't render as bold, if that was your intent. You'll need to pass in subs to make that render properly.)

I'm just wondering if there's a way we can do it that avoids this issue all together. Could we just show the output perhaps w/o the command? That's really the only thing we're trying to convey here. Thoughts?

Reply:
I think that we should show output without the command, because RHEL and OpenShift are doing this so differently.